### ISSUE TITLE
Support for i18n

### ISSUE THREAD

**Issue by webysther** [phpmd/phpmd#367]
On directory [rulesets](/phpmd/phpmd/tree/master/src/main/resources/rulesets) support possibility to add another language, like  `pt-br`, `en`, `es` etc... And to use, add option `--lang pt-br`. Default is `en`.

**Comment by ravage84** [comment]
Hi @Webysther  do you have a proposal how to implement that?

**Comment by Webysther** [comment]
Not clear at all, but i fork and create one, ok?

**Comment by ravage84** [comment]
Yes, please fork PHPMD and create a pull request. We can have a look at your implementation proposal then.
